### PR TITLE
Adjusted 'specification' instead of 'schedule'

### PR DESCRIPTION
**Pull request by valdensmith**
Please see issue #2241

### CODE REVIEW

File: documentation/source/eligible-collateral-representation.rst [hunk 1]
```diff
@@ -209,7 +209,7 @@ or not.
 The combination of these terms allows a wide variety of eligible
 collateral types to be represented and structure can be used to identify
 individual collateral types or a group of collateral assets for
-inclusion in an eligible collateral schedule.
+inclusion in specifying eligible collateral schedule details.
```
Comment (lolabeis) [inline documentation/source/eligible-collateral-representation.rst:212]: I assume you also want to remove 'schedule', i.e. just write 'in specifying eligible collateral details'?

**Review by lolabeis — COMMENTED** [review]
Some minor comment.

Also housekeeping note: you should raise a GitHub issue first, explaining the rationale for the name change (that was a requirement from the WG), and then link your PR to that issue. In truth the issue should also be linked to this previously released change:
https://github.com/finos/common-domain-model/pull/2090

Also make sure you label the issue / PR appropriately based on new process.

File: documentation/source/eligible-collateral-representation.rst (hunk 1 above)
Comment (valdensmith) [reply documentation/source/eligible-collateral-representation.rst:212]: just to clarify did the graphic image upload get fixed too?

Comment (lolabeis) [reply documentation/source/eligible-collateral-representation.rst:212]: Yes it did

Comment (valdensmith) [reply documentation/source/eligible-collateral-representation.rst:212]: On taking second look this looks ok, it describes one of the use cases which is including details in a eligible collateral schedule, so this is ok

Comment (valdensmith) [reply documentation/source/eligible-collateral-representation.rst:212]: Can we proceed with the contribution now to replace the document, thank you

**Review by lolabeis — APPROVED** [review]

**Review by hugohills-regnosys — APPROVED** [review]

**Pull request merged**